core/src/main/resources/lib/form/dropdownList.jelly

```diff
@@ -46,9 +46,9 @@ THE SOFTWARE.
         <j:if test="${attrs.help!=null}">
             <f:helpArea/>
         </j:if>
-        <div class="setting-main">
+        <div class="jenkins-select">
             <!-- create drop-down list -->
-            <select class="setting-input dropdownList"
+            <select class="jenkins-select__input dropdownList"
```

Comment:
credentials widget is just broken in general,

random arrow on the right hand side, button moved to the next line

before:

![image](https://user-images.githubusercontent.com/21194782/142162748-05f660cd-71e9-4c69-b51a-d0f243c4dc90.png)


after:

![image](https://user-images.githubusercontent.com/21194782/142162717-00fde09d-6117-446e-9c40-b010673840f0.png)